src/styles/blocks/heder.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+
+  &__picture {
```

Comment:
As per the checklist, class names should be written in lower case and separated by hyphens. So, `&__picture` should be renamed to `&__picture-style` to follow the BEM naming convention.